example/ck_tile/20_grouped_convolution/grouped_convolution_backward_weight.cpp

```diff
@@ -14,29 +14,176 @@
 #include "grouped_convolution_backward_weight_invoker.hpp"
 #include "run_grouped_convolution_bwd_weight_example.inc"
 
-template <typename GemmWarpConfig>
+template <typename GemmWarpConfig, typename GemmTileConfig, typename GemmVectorLoads>
 int run_grouped_conv_bwd_weight_example(ck_tile::ArgParser& arg_parser)
 {
     using Invoker = GroupedConvolutionBackwardWeightInvoker;
 
-    std::string data_type  = arg_parser.get_str("prec");
-    std::string in_layout  = arg_parser.get_str("in_layout");
-    std::string wei_layout = arg_parser.get_str("wei_layout");
-    std::string out_layout = arg_parser.get_str("out_layout");
+    std::string data_type                = arg_parser.get_str("prec");
+    std::string in_layout                = arg_parser.get_str("in_layout");
+    std::string wei_layout               = arg_parser.get_str("wei_layout");
+    std::string out_layout               = arg_parser.get_str("out_layout");
+    ck_tile::index_t num_groups_to_merge = arg_parser.get_int("num_groups_to_merge");
 
     if(data_type == "fp16")
     {
-        return run_grouped_conv_bwd_weight_example_prec_type<Invoker,
-                                                             GemmWarpConfig,
-                                                             ck_tile::half_t>(
-            in_layout, wei_layout, out_layout, arg_parser);
+        // The undefined (negative) value corresponds to the number of
+        // merged groups equal to unity, i.e., no merged groups.
+        if(num_groups_to_merge <= 1)
+        {
```

Comment:
I dont think that we need all of these configs in the example. We should add them into tile engine but I think one example is enough at now